### ISSUE TITLE
Can we separate the family ID values into a single source of truth

### ISSUE THREAD

**Issue by bcr** [microsoft/uf2#46]
https://github.com/microsoft/uf2/blob/d4a345a1dab584ed805af30820273ca85c58a2a0/utils/uf2conv.py#L16

I imagine things will get out of hand with naming and numbering conventions for the family IDs. I recommend that we have a single location for all of the family IDs (a JSON file perhaps), and some discipline around naming.

Even between the `uf2conv.py` script and the `README.md` for this repo, we're already starting to see some differences (in my example, `Microchip (Atmel) SAMD21` vs. `SAMD21`)

This is going to get even harder when we start trying to be smarter about which family ID matches which board ID. In the example given in the README:

`Board-ID: SAMD21G18A-Zero-v0`

Which family ID will match that? Do we have to create an additional registry that maps between board IDs and family IDs?

**Comment by bcr** [comment]
Between the `README.md` and `uf2conv.py` scripts I found the following discrepancies. I am preparing a patch to help mitigate this.

```
! 0xada52840 was not in Python
! 0xc47e5767 was not in Python
! 0xd42ba06c was not in Python
! 0x7eab61ed was not in Python
! 0x1c5f21b0 was not in Python
! 0x7f83e793 was not in Python
! 0x6d0922fa was not in Python
! 0x5a18069b was not in Python
! 0x8fb060fe was not in Python
! 0x1b57745f was not in README
```

**Comment by jacobrosenthal** [comment]
Any interest in publishing something like this separately or hosting it out of the repo? I maintain [hf2-rs](https://github.com/jacobrosenthal/hf2-rs/) and I have to copy all these in every few months

**Comment by bcr** [comment]
It doesn't matter to me where this lives, happy to help where I can, and happy to take ownership. My dream is it's in a fixed location with unsurprising HTTP caching semantics, which I think is automatic with raw GitHub content.

**Comment by bcr** [comment]
@jacobrosenthal this should be live now at https://raw.githubusercontent.com/microsoft/uf2/master/utils/uf2families.json -- fixed with #47
